VirtoCommerce.LiquidThemeEngine/Filters/FeatureFilter.cs

```diff
@@ -0,0 +1,65 @@
+namespace VirtoCommerce.LiquidThemeEngine.Filters
+{
+    using System;
```

Comment:
Could you do not use this nested `using` declaration? This syntax looks to be inconsistent with our general code style convention. We have negotiated regarding this style in one of dev meetings and have come to an agreement to do not use nested `using` statements.

Reply:
done